.circleci/src/workflows/commit.yml

```diff
@@ -8,6 +8,12 @@ jobs:
   - check_web:
       context:
         - mattermost
+  - e2e_web:
```

Comment:
Shouldn't we execute this for other workflows as well?

Reply:
If you did not this anywhere else because no tests for web are run there already, I think it would be good to add a todo there/add this as well but as comments.